.eslintrc.js

```diff
@@ -113,7 +114,8 @@ module.exports = {
                 },
             },
         ],
-        'import/no-cycle': 'error',
+        // Could be useful, but it's very very very slow
+        'import/no-cycle': 'off',
```

Comment:
Not sure this is wanted. It saved me quite few times.

Reply:
Explain in PR description

Reply:
My bad. I wrote the comment before and didn't notice description was updated.